### ISSUE TITLE
Error invoking remote method 'send-http-request': AggregateError

### ISSUE THREAD

**Issue by Core2023-bit** [usebruno/bruno#3243]
When trying to get the request and response payload, getting an error message stating:
Error invoking remote method 'send-http-request': AggregateError

![image](https://github.com/user-attachments/assets/ee75a34a-c825-4473-82ff-5c41b830bcb1)

Already disabled this: SSL/TLS Certificate Verification, my network is working fine.

**Comment by helloanoop** [comment]
@Core2023-bit 

Is this issue still occurring in the latest version of Bruno, v1.34.0? If it is, we could use some help in better understanding the error. Was it working previously, and did this start happening after an upgrade?

Also, if you're behind a corporate proxy, you may need to update the proxy settings in the preferences.

**Comment by Core2023-bit** [comment]
@helloanoop , this is my first time using bruno. Even on the new version error received. Let me check updating the proxy settings.

**Comment by earshinov** [comment]
Same issue for me, Bruno v1.34.0. Happens on some requests only, but consistently. Also notable, before failing with "...Aggregate error", the request takes a lot of time - around 20 seconds while the same request in Postman completes in 500ms. Is there any way to debug what is happening in Bruno's internals and/or provide you with more information for debugging?

**Comment by Core2023-bit** [comment]
Hi Team, wanted to follow up if you could assist me on this. Thank you

**Comment by Krimarcu** [comment]
I see the same issue, for some days now. It was working well before and suddenly showed the error message.
Bruno was not updated inbetween. I updated now to 1.38.1, but nothing changed.

**Comment by W-Floyd** [comment]
Also facing this issue, I find that it works when I'm on home WiFi, but throws this issue when connected via a mobile hotspot. Web browsers work in either case.

**Comment by andrei-korshikov** [comment]
> you may need to update the proxy settings in the preferences

Yeah, that was exactly my case: I killed proxy and forgot to change preferences.

My complaint is: the error is very uninformative. I think in case of proxy problems it should clearly state: "Can not connect to the proxy." (I'm on **1.38.1**).

**Comment by antonio-fasolato** [comment]
I also had this issue. In my case it was not related to a proxy setting (I une none), but I was accidentally starting my server with SSL enabled and I was using http in Bruno.

For reference, trying to call the same endpoint with curl gave me this result: `curl: (52) Empty reply from server`. I switched to https also in Bruno and obviously the endpoint was called correctly.

**Comment by sreelakshmi-bruno** [comment]
Hi, is it still happening for v3.2.0?

**Comment by anusree-bruno** [comment]
Closing due to inactivity. Please reopen if you are still facing the issue.
